src/test/java/uk/gov/pay/connector/gateway/processor/ChargeNotificationProcessorTest.java

```diff
@@ -53,7 +48,9 @@
     
     @Before
     public void setUp() {
-        gatewayAccount = new GatewayAccountEntity("sandbox", new HashMap<>(), TEST);
+        gatewayAccount = GatewayAccountEntityFixture
+                .aGatewayAccountEntity()
+                .build();
```

Comment:
Could set ID on fixture
```suggestion
        gatewayAccount = GatewayAccountEntityFixture
                .aGatewayAccountEntity()
                .withId(GATEWAY_ACCOUNT_ID)
                .build();
```